R/model.R

```diff
@@ -110,7 +111,7 @@ print.keyATM_docs <- function(x, ...)
 
 
 #' @noRd
-#' @method summary keyATM_docs
+#' @method summarize keyATM_docs
```

Comment:
If you specify S3, I think it should be the same as the name of the function. But it seems we do not need this line: http://r-pkgs.had.co.nz/man.html
I will fix it.

Reply:
Thank you so much. I haven't realized this.
\\